### PR TITLE
feat(env-creation): add processing-timeout option

### PR DESCRIPTION
**Pull request by ruderngespra**
Add optional option to provide a specified timeout in minutes for `--await-processing` upon environment creation.

**Open Issues**
- We should add a test [here](https://github.com/contentful/contentful-cli/blob/master/test/unit/cmds/space_cmds/environment_cmds/create.test.js) that checks that `environmentCreate` triggers `checkAwaitProcessing` with the correct parameters. How to mock that correctly?
- Snapshot tests behave flaky locally.

### CODE REVIEW

**Review by Vinz93 — APPROVED** [review]
LGTM :)

File: lib/cmds/space_cmds/environment_cmds/create.js [hunk 1]
```diff
@@ -88,35 +124,12 @@ async function environmentCreate({
...
-        } else {
-          logging.error('Environment creation failed')
-        }
-        break
-      }
-
-      await Promise.delay(DELAY)
-      count++
-    }
+    checkAwaitProcessing(space, environment, processingTimeout)
```
Comment (makinwab) [inline lib/cmds/space_cmds/environment_cmds/create.js:132]: We should log some type of timeout-related message for users after awaiting processing in the case where the environment is not ready. This way the user knows this and can bump their timeout values.

Comment (Vinz93) [reply lib/cmds/space_cmds/environment_cmds/create.js:132]: Thank you for the feedback, I'll this change in another PR 👍

**Comment by Nintendot** [comment]
🎉 💥 !

**Comment by Nintendot** [comment]
Would this PR open doors for addressing https://github.com/contentful/contentful-cli/issues/831?

**Comment by Vinz93** [comment]
> Would this PR close #831?

Yes! 🙂

**Pull request merged**